### PR TITLE
Components

### PR DESCRIPTION
**Pull request by jasondstrunk**

### CODE REVIEW

**Comment by jasondstrunk** [comment]
@Bouland95

File: Client/App.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      .then(response => {
+        return response.json();
+      })
+      .then(data => {
+        this.setState({
+          allData: data,
+          sortOptions: ['most recent', 'highest rated', 'lowest rated', 'most helpful'],
+          filterOptions: ['5 stars', '4 stars', '3 stars', '2 stars', '1 star']
+        })
+
```
Comment (azizbouland) [inline Client/App.jsx:32]: Unnecessary newline

File: Client/Components/List.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <ul>{props.allData.map(function(element, index) {
+      return <SingleReview
+        key={index}
+        element={element}
+
+      />
+    })}</ul>
+  )
+
+}
```
Comment (azizbouland) [inline Client/Components/List.jsx:16]: Unnecessary newlines at 11 and 15

File: Client/Components/Overview.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Overview = (props) => {
+
+  return (
+    <h3>Guest Rating's and Reviews</h3>
+  )
+
```
Comment (azizbouland) [inline Client/Components/Overview.jsx:8]: Unnecessary newline

File: Client/Components/Sort.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+const Sort = (props) => {
+
+  // let defaultOption = props.sortOptions[0];
+
+
+  return (
+    <div>DROP DOWN MENU GOES HERE</div>
+  )
+
+}
```
Comment (azizbouland) [inline Client/Components/Sort.jsx]: Unnecessary newlines at 10 and 14

File: Server/server.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+
+app.get('/', (req, res) => {
+  res.send()
+});
+
```
Comment (azizbouland) [inline Server/server.js]: is this still doing anything? Since you're serving files through express.static

File: Server/server.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+})
+
+app.get('/api/reviews/:id', (req, res) => {
+  Reviews.findById(req.params.id, function(err, result) {
+    if (err) {
+      console.log('ERROR')
+    } else {
+      res.send(result)
+    }
+  })
```
Comment (azizbouland) [inline Server/server.js]: This will still execute fully if there is an error, you can throw error at line 34 to stop code execution.

**Review by azizbouland — APPROVED** [review]
Looking good! mostly a few formatting comments.

**Pull request merged**